libs/components/src/lib/tooltip/flexible-connected-position-strategy-2.ts

```diff
@@ -34,17 +34,19 @@ FlexibleConnectedPositionStrategy2.prototype['_getNarrowedViewportRect'] =
     // the problem: the viewport changes on mobile browsers depending the scroll direction
     // window innerHeight/innerWidth correctly handles this changes but does not consider scrollbars
     // documentElement clientHeight/clientWidth correctly handles the scrollbars but not the viewport changes
-    //
     // therefore we use `clientWidth` for the width and `innerHeight` for the height
     // this works for use because we never have horizontal scrollbars
 
-    // @ts-expect-error
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
should not be necessary when using @ts-expect-error, but make sure a description is required with @ts-expect-error.